### ISSUE TITLE
Vendors

### ISSUE THREAD

**Issue by justindra** [justindra/townhub#22]
The vendors module is all about adding vendors into the app. Vendors is basically any other business that sells items.

Categories:
* Artisanal Items - Sells locally made arts and crafts
* Food - restaurants, cafes etc.

Initial features:
* All vendors should be listed in order
* Sub-categories or tags?
* Information shown for each vendor:
  * Logo
  * Name
  * Description
  * Tags?
  * Go to website/online store
  * Extra images

Future features:
* Add search - not sure what it would search on, maybe those tags? Add ES text searching through the descriptions?
* Allow vendors to edit their own data at any time, requires a vendor dashboard - maybe use Ruru dashboard to do this and we merge it together?
* Allow vendors to add items/menus to their profile
* Allow vendors to receive orders from here
  * needs payment, square or stripe or both
  * Need to build some sort of sign up thing for customers
  * Basically build an e-commerce side to this

**Comment by justindra** [comment]
Currently, sent out emails to some local artists around Revelstoke to see if this is actually worth building or not.

**Comment by justindra** [comment]
Idea: Use Typeform or Google Sheets to get initial dataset for Vendors
